src/actions/actionTypes.js

```diff
@@ -294,3 +294,6 @@ export const RELEASES_LOAD_ERROR = 'RELEASES_LOAD_ERROR';
 export const RELEASE_SELECTED = 'RELEASE_SELECTED';
 
 export const ROUTER_LOCATION_CHANGE = '@@router/LOCATION_CHANGE';
+
+// Error store management
+export const SINGLE_ERROR_CLEAR = 'SINGLE_ERROR_CLEAR';
```

Comment:
I called it "SINGLE_ERROR_CLEAR" because otherwise calling it the more straightforward "CLEAR_ERROR" matches with the request / error reducers that are looking for things that end with "_ERROR"